cape_encrypt/README.md

```diff
@@ -0,0 +1,23 @@
+Cape Encrypt
+===========
+
+Ease of use utility to allow functions deployed with [Cape](https://docs.capeprivacy.com/getting-started) to leverage enclave only encryption and decryption. These operations are completed using your account level [Cape Key](https://docs.capeprivacy.com/concepts/encrypt).
+
+
+## Installation 
+```sh
+pip install cape_encrypt
```

Comment:
eventually will have to add some github action logic to handle releasing/publishing, but it's fine to do that later

Reply:
added main github workflow but holding off on release until it's an actual registered module